plugins/generator-1.18.2/forge-1.18.2/templates/json/item_cmodel.json.ftl

```diff
@@ -1,11 +1,45 @@
 {
+<#if var_item?? && var_item=="helmet">
+  "parent": "${modid}:custom/${data.helmetItemCustomModelName.split(":")[0]}",

```

Comment:
I'd add some helper methods to the Armor class, such as `getItemCustomModelNameFor(String part)`, so that the model json templates only use a single if-else. By passing `var_item` to the helper methods, you don't have to check each possible case in the .ftl files